ci-operator/step-registry/optional-operators/cvp-common/apply-secrets/optional-operators-cvp-common-apply-secrets-commands.sh

```diff
@@ -40,32 +33,39 @@ EOF
     )
 fi
 
-# Creating file that contains namespace name
-echo "$OO_INSTALL_NAMESPACE" > "${SHARED_DIR}"/operator-install-namespace.txt
 
-GPG_KEY='/var/run/cvp-pyxis-gpg-secret/cvp-gpg.key' # Secret file which will be mounted by DPTP
-GPG_PASS='/var/run/cvp-pyxis-gpg-secret/cvp-gpg.pass' # Secret file which will be mounted by DPTP
-PKCS12_CERT='/var/run/cvp-pyxis-gpg-secret/cvp-dptp.cert' # Secret file which will be mounted by DPTP
-PKCS12_KEY='/var/run/cvp-pyxis-gpg-secret/cvp-dptp.key' # Secret file which will be mounted by DPTP
+# Check if PYXIS_URL exists, skip the whole step if not.
+if [[ -z "$PYXIS_URL" ]]; then
+    echo "PYXIS_URL is not defined!"
+else
+    echo "PYXIS_URL is defined, proceeding with cvp-common-apply-secrets from PYXIS"
+    # Creating file that contains namespace name
+    echo "$OO_INSTALL_NAMESPACE" > "${SHARED_DIR}"/operator-install-namespace.txt
 
-echo "Fetching the kube_objects from Pyxis for ISV pid ${PYXIS_URL}"
-touch /tmp/get_kubeObjects.txt
-curl --key "${PKCS12_KEY}" --cert "${PKCS12_CERT}" "${PYXIS_URL}" | jq -r ".container.kube_objects" > /tmp/get_kubeObjects.txt
+    GPG_KEY='/var/run/cvp-pyxis-gpg-secret/cvp-gpg.key' # Secret file which will be mounted by DPTP
+    GPG_PASS='/var/run/cvp-pyxis-gpg-secret/cvp-gpg.pass' # Secret file which will be mounted by DPTP
+    PKCS12_CERT='/var/run/cvp-pyxis-gpg-secret/cvp-dptp.cert' # Secret file which will be mounted by DPTP
+    PKCS12_KEY='/var/run/cvp-pyxis-gpg-secret/cvp-dptp.key' # Secret file which will be mounted by DPTP
 
-echo "Decrypting the kube_objects fetched from Pyxis"
-gpg --batch --yes --quiet --pinentry-mode loopback --import --passphrase-file "${GPG_PASS}" "${GPG_KEY}"
-gpg --batch --yes --quiet --pinentry-mode loopback --decrypt --passphrase-file "${GPG_PASS}" /tmp/get_kubeObjects.txt > /tmp/kube_objects.yaml
+    echo "Fetching the kube_objects from Pyxis for ISV pid ${PYXIS_URL}"
```

Comment:
I thought that this was for Red Hat operator bundles?

Reply:
Apparently they are not @14rcole 
Infact, If you see, If there is no Pyxis url before. 
The whole step used skip.